.gitignore

```diff
@@ -11,3 +11,4 @@ __pycache__
 venv/
 docs/modules/
 docs/_build/
+.idea
```

Comment:
By that logic, `.swp` shouldn't be in .gitignore either. Similarly, `venv/` might also not fit, since it's not language-specific (i.e. it's not an artifact that happens by compiling or executing the language). 

Overall, I don't mind either way. If you insist, I'll remove the line. However, the change seems to be in accordance to the current gitignore contents to me :)

Reply:
I think having `.idea` here is fine.

Reply:
That sounds like 'broken window theory' to me. I'd be ok with also deleting `.swp` and `venv` (Don't remember if this was produced by anything in the Makefile at any time.).
Since they are already there, we should have more consensus here.
@theforeman/ansible-modules

Reply:
There is also `.mypy_cache`, but we don't use mypy here ;)